crates/egui/src/widgets/drag_value.rs

```diff
@@ -490,7 +490,10 @@ impl<'a> Widget for DragValue<'a> {
                     .clip_text(false)
                     .horizontal_align(ui.layout().horizontal_align())
                     .vertical_align(ui.layout().vertical_align())
-                    .margin(ui.spacing().button_padding)
+                    .margin(Margin::symmetric(
+                        ui.spacing().button_padding.x,
+                        ui.spacing().button_padding.y,
+                    ))
```

Comment:
`Margin should implement `From<Vec2>` so that the old code still compiles

Reply:
Sure, I can add that.

Reply:
Already exists: https://github.com/emilk/egui/blob/d22ee81ab02a96ff2cfa15903b0d4255bec55962/crates/egui/src/style.rs#L684